c2corg_api/scripts/migration/documents/waypoints/summit.py

```diff
@@ -70,6 +72,13 @@ def get_document_locale(self, document_in, version):
             summary=summary
         )
 
+    def get_climbing_outdoor_types(self, document_in):
+        outdoor_types = []
+        if document_in.summit_type == 4:
+            outdoor_types.append('multi')
+        outdoor_types = outdoor_types if outdoor_types else None
+        return outdoor_types
```

Comment:
> I didn't try understanding why each line has been coded like this.

You should. Understanding what you do is important.

The code you are pointing is much more complex than the one you have written. Then you could have optimized your code as I have suggested it in order to have a simpler and more readable code. Please also squash your commits when creating a PR unless having distinct commits helps understand the changes.

I won't change the code. Not my job to make the clean up.